### DISCUSSION TITLE
How to use --port flag

### DISCUSSION THREAD

**Discussion by KevinYangMay** [simonhaenisch/md-to-pdf#125] [Q&A] [answered]
Hi,I don't know how to use flag， can you provide some code examples，Thanks！ @ @simonhaenisch

**Comment by simonhaenisch** [comment, marked as answer]
Hey, basically like `md-to-pdf --port 12345`. TBH I don't think there's really a use case for it, because the process just finds an available port automatically by default (using https://github.com/sindresorhus/get-port). A port is needed because it spins up a local file server... if you want to know why, i suggest you jump into the source code but basically it's so that the Puppeteer browser instance that is used to generate the PDF can access images via relative paths.

**Reply by KevinYangMay** [reply]
II'll try it right away，Thanks
